### PR TITLE
Fix to minikube cp functional test work on windows os

### PR DESCRIPTION
**Pull request by anencore94**
Bug from PR #10198 

resolve : #10914 

Thanks for finding it @medyagh

### CODE REVIEW

**Comment by medyagh** [comment]
@anencore94 
Checkout the setup certes in the functional. Test to see an example

**Comment by medyagh** [comment]
Checkout other tests copying files 
```
func setupFileSync(ctx context.Context, t *testing.T, profile string) {
	p := localSyncTestPath
...
```

**Comment by anencore94** [comment]
Changed List
- make test code more understandable like `func setupFileSync(ctx context.Context, t *testing.T, profile string) {...}`
- change `filepath.IsAbs()` to `strings.HasPrefix()`.
  - since on windows host os, `filepath.IsAbs('/a/b/c')` equals false
  - I'm a bit confused why this happens..

Please Let me know if there are other issues.
Thanks a lot always, @medyagh

File: test/integration/functional_test.go [hunk 1]
```diff
@@ -1180,30 +1180,47 @@ func validateSSHCmd(ctx context.Context, t *testing.T, profile string) {
...
 // validateCpCmd asserts basic "cp" command functionality
 func validateCpCmd(ctx context.Context, t *testing.T, profile string) {
 	if NoneDriver() {
 		t.Skipf("skipping: cp is unsupported by none driver")
 	}
 
-	cpPath := filepath.Join(*testdataDir, "cp-test.txt")
-	rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "cp", cpPath, "/home/docker/hello_cp.txt"))
+	srcPath := cpTestLocalPath()
+	dstPath := cpTestMinikubePath()
```
Comment (medyagh) [inline test/integration/functional_test.go:1200]: lets not make a func for that, and just simply put it here, easier to read

**Review by medyagh — CHANGES_REQUESTED** [review]

**Pull request merged**